Database/Corrections/sodQuestFixes.lua

```diff
@@ -806,6 +806,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.zoneOrSort] = zoneIDs.GNOMEREGAN,
             [questKeys.questFlags] = questFlags.RAID,
         },
+        [79677] = { -- A Quick Grocery Run
+            [questKeys.startedBy] = {nil,nil,{213422}},
```

Comment:
Questie currently does not handle `item`starters. So with this, the rune simply will not show. Please add the NPC starters for now.